docs/make-changes-to-brexit-checker-content.md

```diff
@@ -50,19 +43,8 @@ bundle exec rake brexit_checker:convert_csv_to_yaml:actions[path/to/actions.csv]
 
 ## Adding change notes
 
```

Comment:
Maybe describing what a change note is and what information it represents would be helpful - I think there might be confusion as they are different from the use of "change note" in the wider GOV.UK publishing platform.

Reply:
Sure. I've updated to give more indication about how and when they're used. I'm trying to avoid talking about specific fields or examples, in case the structure changes - the link to the file should suffice.